netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +159,31 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages has to throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test(expected = HttpException.class)
+    public void exceptionIfBoutNotFound() throws IOException {
```

Comment:
@2686747

1. This never throws these exceptions because you catch them below.
2. We generally just throw `Exception` on tests, see [here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names)